client/Queries/README.md

```diff
@@ -0,0 +1,71 @@
+# Queries
+
+This directory is home to all GraphQL queries. These should be imported by the appropriate models and exposed via a hook.
+
+## File Structure
+
+- Queries
+  - query set (e.g. topics)
+    - index.ts - containing all gql queries for that set
+
+## Example
```

Comment:
Any thoughts on how/if these should be tested? I like the idea of keeping like queries together, but worry a query is changed for one use case, braking another. Is the expectation tests in say the topics page would fail in such a scenario?

Reply:
After some discussion, we've decided that queries shouldn't need testing themselves. The main argument being these could easily be .gql files. Re queries changing: this is a bridge to cross when we get to it although we all have some idea of what could happen when we get there